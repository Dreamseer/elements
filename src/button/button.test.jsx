/**
 * Button Test
 * @author Marc GÃ¶rtz <https://marcgoertz.de/>
 */

import React from 'react';
import renderer from 'react-test-renderer';
import Button from '.';

describe('<Button />', () => {
  it('renders a button', () => {
    const component = renderer.create(<Button />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
