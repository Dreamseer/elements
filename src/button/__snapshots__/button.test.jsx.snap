// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> renders a button 1`] = `
.emotion-0 {
  display: inline-block;
  margin: 0;
  padding: 0.25rem 1rem;
  border: 0;
  font-size: inherit;
  font-family: system-ui,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Helvetica,Arial,sans-serif,'Apple Color Emoji','Segoe UI Emoji','Segoe UI Symbol';
  line-height: inherit;
  color: white;
  background-color: lightgrey;
  box-sizing: border-box;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
  cursor: pointer;
}

.emotion-0:hover,
.emotion-0:focus {
  background-color: #008ec0;
}

<button
  className="emotion-0 emotion-1"
  disabled={false}
  type="button"
/>
`;
